temporal-sdk/src/test/java/io/temporal/worker/LocalActivityWorkerNotStartedTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.worker;
+
+import static org.junit.Assert.assertTrue;
+
+import io.temporal.testing.internal.SDKTestWorkflowRule;
+import io.temporal.workflow.Workflow;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Set;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class LocalActivityWorkerNotStartedTest {
+
+  @Rule
+  public SDKTestWorkflowRule testWorkflowRule =
+      SDKTestWorkflowRule.newBuilder()
+          .setWorkflowTypes(NothingWorkflowImpl.class)
+          .setWorkerOptions(WorkerOptions.newBuilder().setLocalActivityWorkerOnly(true).build())
```

Comment:
Don't you have to register some activity types or else the local activity worker will never start?

Reply:
Or do you think we have good coverage of the started case from existing tests?

Reply:
There are different scenarios right:
* No workers are started
* The workflow worker is started, but the local activity worker is not started

Reply:
The whole point is he doesn't actually start - the test did repro the behavior before the change.

Reply:
Other tests adequately cover the actual started case I believe

Reply:
Right I was saying there are two ways you can create a worker that doesn't start the local activity worker

Reply:
I will add one with an activity registered too :+1: